### DISCUSSION TITLE
Initialize Mikro ORM without defining entities

### DISCUSSION THREAD

**Discussion by hubbusysyuhada** [mikro-orm/mikro-orm#2941] [Q&A] [answered]
I need to initialize a Mikro ORM instance with 0 entities at first and keep adding it more as time goes by.

I tried to use `MikroORM.init({entities: []})`
but it keeps throwing an error that said `('No entities found, please use 'entities' option')`

is there anything that I could do? or the `entities` property must contain at least 1 entities?

**Comment by B4nan** [comment, marked as answer]
Not sure what you expect from that, but you can disable this validation via `discovery: { warnWhenNoEntities: false }` (internally this is used when you want to generate entities from existing schema).

**Reply by hubbusysyuhada** [reply]
great, that's what I've been looking for! Thank you
